public/tailwind.css

```
@@ -721,19 +801,20 @@
-  width: 26px;
-  height: 26px;
-  line-height: 26px;
-  text-align: center;
+  font-size: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 32px;
+  height: 32px;
   border-radius: 50%;
   border: 0;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)